tools/android/packaging/xbmc/build.gradle.in

```diff
@@ -1,26 +1,35 @@
 apply plugin: 'com.android.application'
 
 android {
-    compileSdkVersion 29
-    buildToolsVersion "29.0.3"
+    compileSdkVersion 31
```

Comment:
This seems unrelated to this PR and likely has other impacts. Is there a reason you need to do this here?